arrow-array/src/array/fixed_size_list_array.rs

```diff
@@ -60,6 +113,9 @@ use std::sync::Arc;
 /// assert_eq!( &[3, 4, 5], list1.as_any().downcast_ref::<Int32Array>().unwrap().values());
 /// assert_eq!( &[6, 7, 8], list2.as_any().downcast_ref::<Int32Array>().unwrap().values());
 /// ```
+///
+/// [`StringArray`]: crate::array::StringArray
+/// An array of [fixed size arrays](https://arrow.apache.org/docs/format/Columnar.html#fixed-size-list-layout)
```

Comment:
I think this link formatting is a little off

```suggestion
/// [fixed size arrays]: https://arrow.apache.org/docs/format/Columnar.html#fixed-size-list-layout
```